etna/api/urls.py

```diff
@@ -28,7 +28,7 @@
 
 class CustomPagesAPIViewSet(PagesAPIViewSet):
     known_query_parameters = PagesAPIViewSet.known_query_parameters.union(
-        ["password", "author"]
+        ["password", "author", "include_aliases"]
```

Comment:
Are we guaranteed that `include_aliases` will be the last filter using `.union()`?

We need to ensure it is applied after all other filtering.

Reply:
🤦🏼‍♂️

You are right - sorry!